api/namex/constants/__init__.py

```diff
@@ -271,6 +282,9 @@ class LegacyEntityTypes:
     ('CCR', EntityTypes.CORPORATION.value, RequestAction.RESUBMIT.value),
     ('CT', EntityTypes.CORPORATION.value, RequestAction.MVE.value, True),
     ('CT', EntityTypes.CORPORATION.value, RequestAction.RESUBMIT.value),
+    ('CT', EntityTypes.CONTINUE_IN.value, RequestAction.RESUBMIT.value),
+    ('CT', EntityTypes.CONTINUE_IN.value, RequestAction.REH.value),
+    ('CT', EntityTypes.CONTINUE_IN.value, RequestAction.CHG.value),
```

Comment:
I provided the value of "CT" based on this comment:

![image](https://github.com/user-attachments/assets/cdd32ab9-663e-499c-909a-5d60564846a1)

However, I now wonder if the code for CHG should be "CCR", as on line 281?

Also, I believe all codes have a "RESUBMIT" entry. For example:
- CT, MVR
- CT, RESUBMIT
- CCR, CHG
- CCG, RESUBMIT
- RCR, REH
- RCR, RESUBMIT

Note also that there are RCR codes and REH, REN and REST, and similar in this structure, so I think we should do the same thing.

Reply:
Resolving for now. Will review new code changes instead.